packages/react/src/Tabs/TabsButton.test.tsx

```diff
@@ -52,6 +53,26 @@ describe('Tabs button', () => {
     expect(component).toHaveAttribute('aria-controls', '-panel-0')
   })
 
+  it('calls onChange with the newly activated tab', () => {
```

Comment:
Why is this in `TabsButton.test`? It's a change to `Tabs`, I think it should go in `Tabs.test`